fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/config/jpa/JPAConfig.java

```diff
@@ -70,29 +77,48 @@ public JPAConfig(RoutingDataSource dataSource, JpaProperties properties, ObjectP
     @DependsOn("tenantDatabaseUpgradeService")
     public LocalContainerEntityManagerFactoryBean entityManagerFactory(EntityManagerFactoryBuilder factoryBuilder) {
         Map<String, Object> vendorProperties = getVendorProperties();
-        customizeVendorProperties(vendorProperties);
-        return factoryBuilder.dataSource(getDataSource()).properties(vendorProperties).persistenceUnit("jpa-pu")
-                .packages("org.apache.fineract").jta(false).build();
+        String[] packagesToScan = getPackagesToScan();
+        return factoryBuilder.dataSource(getDataSource()).properties(vendorProperties).persistenceUnit("jpa-pu").packages(packagesToScan)
+                .jta(false).build();
+    }
+
+    @Override
+    protected Map<String, Object> getVendorProperties() {
+        Map<String, Object> vendorProperties = new HashMap<>();
+        vendorProperties.put(PersistenceUnitProperties.WEAVING, "static");
+        vendorProperties.put(PersistenceUnitProperties.PERSISTENCE_CONTEXT_CLOSE_ON_COMMIT, "true");
+        vendorProperties.put(PersistenceUnitProperties.CACHE_SHARED_DEFAULT, "false");
+        emFactoryCustomizers.forEach(c -> vendorProperties.putAll(c.additionalVendorProperties()));
```

Comment:
Same here... adding this properties (with defaults) to application.properties would be easier?